### PR TITLE
Render group chat in multiline format

### PR DESCRIPTION
**Pull request by theRemix**
Display group chats in multiple lines

- improves readability of group chats in channel list

```
│  # channel-1 ││
│  # channel-2 ││
│  ☰ andrea    ││ </-- each of these 4 lines
│    kelli     ││      select the same "andrea, kelli, jon, beerock"
│    jon       ││      group chat
│    beerock   ││
│  ☰ derek     ││
│    jon       ││
```

### CODE REVIEW

File: components/chat.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/erroneousboat/termui"
 	runewidth "github.com/mattn/go-runewidth"
 
-	"github.com/erroneousboat/slack-term/config"
+	"github.com/theremix/slack-term/config"
```
Comment (frzifus) [inline components/chat.go]:
```suggestion
	"github.com/erroneousboat/slack-term/config"
```

File: context/context.go [hunk 2]
```diff
@@ -10,9 +10,9 @@ import (
 	"github.com/erroneousboat/termui"
 	termbox "github.com/nsf/termbox-go"
 
-	"github.com/erroneousboat/slack-term/config"
-	"github.com/erroneousboat/slack-term/service"
-	"github.com/erroneousboat/slack-term/views"
+	"github.com/theremix/slack-term/config"
+	"github.com/theremix/slack-term/service"
+	"github.com/theremix/slack-term/views"
```
Comment (frzifus) [inline context/context.go]: s/theremix/erroneousboat/g

File: handlers/event.go [hunk 3]
```diff
@@ -14,10 +14,10 @@ import (
...
 	termbox "github.com/nsf/termbox-go"
 
-	"github.com/erroneousboat/slack-term/components"
-	"github.com/erroneousboat/slack-term/config"
-	"github.com/erroneousboat/slack-term/context"
-	"github.com/erroneousboat/slack-term/views"
+	"github.com/theremix/slack-term/components"
+	"github.com/theremix/slack-term/config"
+	"github.com/theremix/slack-term/context"
+	"github.com/theremix/slack-term/views"
```
Comment (frzifus) [inline handlers/event.go]: same as above

File: main.go [hunk 4]
```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/erroneousboat/termui"
 	termbox "github.com/nsf/termbox-go"
 
-	"github.com/erroneousboat/slack-term/context"
-	"github.com/erroneousboat/slack-term/handlers"
+	"github.com/theremix/slack-term/context"
+	"github.com/theremix/slack-term/handlers"
```
Comment (frzifus) [inline main.go]: same

File: service/slack.go [hunk 5]
```diff
@@ -12,8 +12,8 @@ import (
 
 	"github.com/nlopes/slack"
 
-	"github.com/erroneousboat/slack-term/components"
-	"github.com/erroneousboat/slack-term/config"
+	"github.com/theremix/slack-term/components"
+	"github.com/theremix/slack-term/config"
```
Comment (frzifus) [inline service/slack.go]: same

File: views/view.go [hunk 6]
```diff
@@ -3,9 +3,9 @@ package views
 import (
 	"github.com/erroneousboat/termui"
 
-	"github.com/erroneousboat/slack-term/components"
-	"github.com/erroneousboat/slack-term/config"
-	"github.com/erroneousboat/slack-term/service"
+	"github.com/theremix/slack-term/components"
+	"github.com/theremix/slack-term/config"
+	"github.com/theremix/slack-term/service"
```
Comment (frzifus) [inline views/view.go]: same

**Comment by erroneousboat** [comment]
Closing this. This is mix-mash of PR's, and at the moment it has conflicts. Also please refer to the CONTRIBUTING.md

**Pull request closed without merging**